package-lock.json

```diff
@@ -10658,10 +10663,37 @@
 				"re-resizable": "^6.0.0",
 				"react-dates": "^17.1.1",
 				"react-spring": "^8.0.20",
-				"reakit": "^1.0.0-beta.12",
+				"reakit": "^1.0.0-rc.0",
 				"rememo": "^3.0.0",
 				"tinycolor2": "^1.4.1",
 				"uuid": "^3.3.2"
+			},
+			"dependencies": {
```

Comment:
you should install `reakit` in the root of the project and then remove it to move it to the root level in the lock file